### PR TITLE
Add extra_hardware and lldp_basic to processing hooks

### PR DESCRIPTION
**Pull request by dtantsur**
These hooks format the raw data from IPA into nicer formats:
* extra_hardware formats the python-hardware inventory from list of lists
  to nested dictionaries under the "extra" key.
* lldp_basic formats the raw LLDP data into human-readable format under
  lldp_processed on each interface.

Related to https://github.com/metal3-io/baremetal-operator/issues/179

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,8 @@ RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noa
     crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
     crudini --set /etc/ironic-inspector/inspector.conf pxe_filter driver noop && \
     crudini --set /etc/ironic-inspector/inspector.conf processing node_not_found_hook enroll && \
-    crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi
+    crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi \
```
Comment (elfosardo) [inline Dockerfile]: I think you're missing "&&" at the end of the line

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (dtantsur) [reply Dockerfile]: Good point. And building images locally doesn't work for me :(

**Review by zaneb — APPROVED** [review]

**Review by elfosardo — APPROVED** [review]

**Comment by dtantsur** [comment]
Do not merge, we need to fix the situation highlighted in the comments on https://review.opendev.org/#/c/658826/

**Review by bfournie — APPROVED** [review]

**Comment by dtantsur** [comment]
This can be merged as soon as https://review.opendev.org/#/c/659213/ hits our repositories.

**Comment by bfournie** [comment]
Verified that  https://review.opendev.org/#/c/659213/ is available in image at https://quay.io/repository/metal3-io/ironic-inspector.

**Pull request merged**